Automatic Validation ended with:
> ##[error] Installer https://releases.r2.labymod.net/launcher/win32/x64/LabyModLauncherSetup-latest.exe
##[error] Result: Failed
##vso[task.logissue type=error] Specified hash doesn't match.
Actual hash:   9274587020718C3AB64B013A0E18822526C53EA687F29840A340920345CB68C4
Expected hash: 6cdca60032796c809a1da444939ff2c01f2345497ecd0d0807ab04aa28e72190

--- a/manifests/l/LabyMediaGmbH/LabyModLauncher/1.0.19/LabyMediaGmbH.LabyModLauncher.installer.yaml
+++ b/manifests/l/LabyMediaGmbH/LabyModLauncher/1.0.19/LabyMediaGmbH.LabyModLauncher.installer.yaml
@@ -7,6 +7,6 @@ Installers:
 - Architecture: x64
   InstallerType: exe
   InstallerUrl: https://releases.r2.labymod.net/launcher/win32/x64/LabyModLauncherSetup-latest.exe
-  InstallerSha256: 9274587020718c3ab64b013a0e18822526c53ea687f29840a340920345cb68c4
+  InstallerSha256: 2A2D8CE635997EAEB04F59DF8C740EAA79BAD5959D998FA698FB2711897C4269
 ManifestType: installer
 ManifestVersion: 1.4.0

--- a/manifests/l/LabyMediaGmbH/LabyModLauncher/1.0.19/LabyMediaGmbH.LabyModLauncher.installer.yaml
+++ b/manifests/l/LabyMediaGmbH/LabyModLauncher/1.0.19/LabyMediaGmbH.LabyModLauncher.installer.yaml
@@ -7,6 +7,6 @@ Installers:
 - Architecture: x64
   InstallerType: exe
   InstallerUrl: https://releases.r2.labymod.net/launcher/win32/x64/LabyModLauncherSetup-latest.exe
-  InstallerSha256: 6CDCA60032796C809A1DA444939FF2C01F2345497ECD0D0807AB04AA28E72190
+  InstallerSha256: 9274587020718c3ab64b013a0e18822526c53ea687f29840a340920345cb68c4
 ManifestType: installer
 ManifestVersion: 1.4.0

